css-basics/css-basics_pokemon-page/css/styles.css

```diff
@@ -58,27 +76,19 @@ section {
 - Bonus: The color of the border of "Favourite Pokémon" should be "orange". Hint: You may need another CSS rule.
 */
 
-.subtitle {
-  /* your solution */
+.favourites .subtitle {
```

Comment:
I think I did it because the section "About my passion" also has the subtitle class but I only wanted to change the .favourite .subtitle class.
But yeah it's quite confusing to me as well.